examples/sample_pipeline/pipeline.py

```diff
@@ -32,8 +35,28 @@
     "./components/dummy_component",
 )
 
-dataset.apply(
+dataset = dataset.apply(
     "chunk_text",
     arguments={"chunk_size": 10, "chunk_overlap": 2},
     consumes={"text": "text_data"},
 )
+
+
+@lightweight_component(
+    base_image="python:3.8",
+    extra_requires=[
+        f"fondant[component]@git+https://github.com/ml6team/fondant@"
+        f"{os.environ.get('FONDANT_VERSION', 'main')}",
```

Comment:
What happens if we incorporate a bug in the actual branch and using the main branch/version to test the lightweight_component? Assume the test will be successful even if it shouldn't be, no? 

Usually, a user doesn't have to addd fondant as an extra requirement?

Reply:
> Usually, a user doesn't have to addd fondant as an extra requirement?

Indeed, this is only needed until we have a Fondant base image.
We might want to keep defining it as an extra requirement in our tests though to test the latest version.

> What happens if we incorporate a bug in the actual branch and using the main branch/version to test the lightweight_component? Assume the test will be successful even if it shouldn't be, no?

In CI/CD, the current branch is being tested. The `FONDANT_VERSION` environment variable is set in the `run.sh` script. I just provided a back-up to run it locally without this setup.